src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -233,3 +236,27 @@ export function getBoxPlotScatterData(molecularData: NumericGeneMolecularData[],
     }
     return scatterData;
 }
+
+export function pickMutationEnrichmentProfiles(profiles:MolecularProfile[]) {
+    return _.filter(profiles, (profile: MolecularProfile) =>
+        profile.molecularAlterationType === AlterationTypeConstants.MUTATION_EXTENDED);
+}
+
+export function pickCopyNumberEnrichmentProfiles(profiles:MolecularProfile[]) {
+    return _.filter(profiles, (profile: MolecularProfile) =>
+        profile.molecularAlterationType === AlterationTypeConstants.COPY_NUMBER_ALTERATION && profile.datatype === "DISCRETE");
+}
+
+export function pickMRNAEnrichmentProfiles(profiles:MolecularProfile[]) {
+    const mrnaProfiles = profiles.filter(p=>{
+        return p.molecularAlterationType === AlterationTypeConstants.MRNA_EXPRESSION
+    });
+    return Promise.resolve(filterAndSortProfiles(mrnaProfiles));
```

Comment:
may be just send the result instead of promise

Reply:
oops good catch